packages/components/forms/src/Multiselect/MultiselectOption.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Checkbox } from '../Checkbox';
+import { Text } from '@contentful/f36-typography';
+import { getMultiselectStyles } from './Multiselect.styles';
+import { getStringMatch } from '@contentful/f36-utils';
+
+export interface MulitselectOptionProps {
+  label: string;
+  value: string;
+  itemId: string;
+  searchValue?: string;
+  onSelectItem: (event: React.ChangeEvent) => void;
+  isChecked?: boolean;
+  isDisabled?: boolean;
+}
+
+export const MultiselectOption = ({
+  label,
+  value,
+  itemId,
+  onSelectItem,
+  searchValue,
+  isChecked = false,
+  isDisabled = false,
+  ...rest
+}: MulitselectOptionProps) => {
+  const styles = getMultiselectStyles();
+
+  return (
+    <li {...rest}>
+      <label htmlFor={itemId} className={styles.item(isDisabled)}>
```

Comment:
You can probably add styles.disabled instead of rewriting the disabled styles in styles.item

Reply:
I don t understand the difference here, can you explain to me what is the benefit of combining two classes vs changing the style via the variable?  because i have seen both patterns across components.

Reply:
Combining classes will allow you to remove `opacity` and `cursor` from `item` styles because they already exist in `disabled` and avoid duplications

Reply:
Is it also a possibility to do something like `input[disabled] + label` instead of doing it via JS?